Source/Controllers/MainViewControllers/SPCustomQuery.m

```diff
@@ -2497,11 +2500,6 @@ - (BOOL)tableView:(NSTableView *)aTableView shouldEditTableColumn:(NSTableColumn
     // Check if the field can identified bijectively
     if ( aTableView == customQueryView ) {
         
-        // Nothing is editable while the field editor is running.
-        // This guards against a special case where accessibility services might
-        // check if a table field is editable while the sheet is running.
-        if (fieldEditor) return NO;
```

Comment:
Scares me but I trust ya if this has been tested lol

Reply:
This is ARC thing - previously, the field editor was nil when closed, but it's not now as we hold it as a property. So once you opened the editor once, you never could open it again...